UnityProject/Assets/Scripts/Items/Medical/Implanter.cs

```diff
@@ -0,0 +1,137 @@
+using UnityEngine;
+using HealthV2;
+using Items;
+
+public class Implanter : MonoBehaviour, ICheckedInteractable<HandApply>, ICheckedInteractable<InventoryApply>, ICheckedInteractable<HandActivate>
+{
+	[SerializeField]
+	private GameObject implantObject = null;
+
+	[SerializeField]
+	private int timeToImplant = 5;
+
+	[SerializeField]
+	private ItemTrait ImplantableTrait = null;
+
+	[SerializeField]
+	private SpriteHandler spriteHandler;
+
+	private ItemStorage itemStorage;
+	private bool primed = false;
+
+	private void Awake()
+	{
+		itemStorage = GetComponent<ItemStorage>();
+	}
+
+	public void OnSpawnServer(SpawnInfo info)
+	{
+		if (implantObject != null)
+		{
+			Inventory.ServerSpawnPrefab(implantObject, itemStorage.GetIndexedItemSlot(0));
+		}
+	}
+
+	public bool WillInteract(HandApply interaction, NetworkSide side)
+	{
+		if (!DefaultWillInteract.Default(interaction, side)) return false;
+		//can only be applied to LHB
+		if (!Validations.HasComponent<LivingHealthMasterBase>(interaction.TargetObject)) return false;
+
+		if (interaction.Intent != Intent.Help || primed == false) return false;
+
+		if(itemStorage.GetIndexedItemSlot(0).IsEmpty)
+		{
+			Chat.AddExamineMsgFromServer(interaction.Performer, "No implant present in implanter!");
+			return false;
+		}
+
+		return true;
+	}
+
+	public void ServerPerformInteraction(HandApply interaction)
+	{
+		var LHB = interaction.TargetObject.GetComponent<LivingHealthMasterBase>();
+
+		foreach (var bodyPart in LHB.SurfaceBodyParts)
+		{
+			bool selected = false;
+
+			if (bodyPart.BodyPartType == interaction.TargetBodyPart)
+			{
+				ItemSlot toSlot = bodyPart.OrganStorage.GetNextFreeIndexedSlot();
```

Comment:
the only issue with this is you can technically implant into anything, as long as its surface body part

Reply:
I get that isn't intended. Why not?

Reply:
being able to implant stuff in people's arms for example and then not being able to remove it

Reply:
because I don't think there's any open surgery option on stuff like arms

Reply:
so what's your suggestion? To move this pr forward

Reply:
haven't been on github in awhile, but as for the concern with legs and arms and such, that will only be the case if an implant is added that can be implanted into arms and legs, now there is a change i need to make because I am now realising i set something up wrong so ill need to do another commit. so yes techincally its possible to implant into arms and not be able to remove it, but thatll oinly happen if someone adds an arm implant without a remove procedure

Reply:
cool sounds good